actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -11,7 +11,7 @@ class Configuration
       def initialize
         @log_tags = []
 
-        @connection_class = -> { ActionCable::Connection::Base }
+        @connection_class = -> { ActionCable::Client::Base }
```

Comment:
Should we keep `connection_class` around, and introduce a new config option called `client_class` or similar? Otherwise in `ActionCable::Server::Base#call` we are declaring an explicit dependency on `ActionCable::Connection::Base`.